CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### Bug fixes
+
+* [#9208](https://github.com/rubocop-hq/rubocop/issues/9208): Use Array#bsearch instead of Array#include? to detect hidden files. ([@dark-panda][])
+
```

Comment:
Please use `rake changelog:change` (or `changelog:fix`, but I think this makes more sense as a change) instead of directly editing the CHANGELOG.